Cubical/Categories/Bifunctor/Base.agda

```diff
@@ -0,0 +1,239 @@
+{-
+
+  Explicit definition of a bifunctor as a "separately functorial"
+  action satisfying a commutativity condition. This is equivalent to
+  defining a bifunctor as a functor out of a product, but has better
+  definitional behavior in some cases.
+
+-}
+
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Bifunctor.Base where
+
+open import Cubical.Foundations.Prelude hiding (Path)
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Constructions.BinProduct hiding (Fst; Snd)
+open import Cubical.Categories.Instances.Sets
+open import Cubical.Categories.Instances.Functors
+
+open import Cubical.Data.Sigma
+open import Cubical.Tactics.CategorySolver.Reflection
+
+private
+  variable
+    ℓc ℓc' ℓd ℓd' ℓe ℓe' : Level
+
+open Category
+open Functor
+open NatTrans
+
+record Bifunctor (C : Category ℓc ℓc')
```

Comment:
This definition is subsumed by `BifunctorSep` in `Bifunctor.Redundant`. I don't think we need *this* file at all, and we should just include `Bifunctor.Redundant` as `Bifunctor.Base`.